src/AzureStack/Admin/ComputeAdmin/Compute.Admin.Tests/Compute.Tests.csproj

```diff
@@ -6,9 +6,9 @@
     <AssemblyName>Compute.Admin.Tests</AssemblyName>
     <VersionPrefix>1.0.0</VersionPrefix>    
   </PropertyGroup>
-  <!--<PropertyGroup>
```

Comment:
Please remove this change. Our tests are now running against netcoreapp2.0

Reply:
You are saying they should leave this commented out?  If so they should just delete this XML entry then.

Reply:
Leave it commented for now, we will do a cleanup throughout the repo at a later point.

Reply:
OK. I will leave it as what it is.

Reply:
@dsgouda I change it back. thanks.